recipe/meta.yaml

```diff
@@ -50,12 +52,17 @@ outputs:
 
     requirements:
       host:
-        - python >=3.6
+        - python >=3.8
       run:
-        - python >=3.6
+        - python >=3.8
+        - openmm >=7.6
+        - openff-interchange =0.2
```

Comment:
Is there a plan to allow this to be installed alongside the newer version?

Reply:
Nah, unintentional. I'll bump Interchange and Python for the 0.13 release. I can't recall a reason to bump openmm to >=8, could you remind me if there is one?

Reply:
No compelling reason and some users might still be using things that require 7 (I think https://github.com/andysim/MPIDOpenMMPlugin is not updated, for example). We should probably communicate such a change before making it